test/controllers/api/v2/smart_proxies_controller_test.rb

```diff
@@ -102,12 +102,14 @@ def setup_import_classes
       Puppetclass.destroy_all
       Environment.destroy_all
     end
+    orgs = [taxonomies(:organization1)]
```

Comment:
This code is duplicated on both API controllers, can you extract it? (see test/controllers/shared/ for examples)